xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -59,10 +59,23 @@ void CGUIWindowPVRGuide::OnInitWindow()
 
   CGUIEPGGridContainer *epgGridContainer =
     dynamic_cast<CGUIEPGGridContainer*>(GetControl(m_viewControl.GetCurrentControl()));
+
   if (epgGridContainer)
-    epgGridContainer->GoToNow();
+  {
+    CPVRChannelGroupPtr group(GetGroup());
+    if (!group)
+      group = g_PVRManager.GetPlayingGroup(m_bRadio);
```

Comment:
Call to CGUIWindowPVRBase::OnInitWindow will call CGUIMediaWindow::OnInitWindow which triggers CGUIWindowPVRGuide::Update(). For the latter to work some values of CGUIWindowPVR[Base|Guide] need to be properly initialized in beforehand (grid start end time, esp. and for this group is needed), otherwise the first Update() will fail with the visual result of this bug. Currently, the first Update() may fail and following work by luck (using the values from the Update before). It's kinda complex, but I debugged it step by step.

So, moving CGUIWindowPVRBase::OnInitWindow to the top of CGUIWindowPVRGuide::OnInitWindow will unfortunately reintroduce the bug.